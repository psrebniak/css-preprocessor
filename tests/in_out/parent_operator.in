parent {
    & child {
        test: test;
    }
    &_child {
        test: test;
    }
    grandparent & {
        test: test;
    }
    grandparent_& {
        test: test;
    }
    grandfather &, child {
        test: test;
    }
}
