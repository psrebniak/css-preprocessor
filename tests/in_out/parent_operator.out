parent child {
    test: test;
}
parent_child {
    test: test;
}
grandparent parent {
    test: test;
}
grandparent_parent {
    test: test;
}
